### PR TITLE
data:KishanGupta

### PR DESCRIPTION
**Pull request by KishanGupta114**

### CODE REVIEW

File: data/KishanGupta.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/i_kishan_",
+      "icon": "FaTwitter"
+    }
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/i-kishan",
+      "icon": "FaLinkedIn"
+    }
```
Comment (SurajPratap10) [inline data/KishanGupta.json:15]:
```suggestion
  "links": [
    {
      "name": "Follow me on Twitter",
      "url": "https://twitter.com/i_kishan_",
      "icon": "FaTwitter"
    },
    {
      "name": "Follow me on LinkedIn",
      "url": "https://www.linkedin.com/in/i-kishan",
      "icon": "FaLinkedIn"
    },
```

Comment (SurajPratap10) [reply data/KishanGupta.json:15]: `,`

File: data/KishanGupta.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/i_kishan_",
+      "icon": "FaTwitter"
+    }
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/i-kishan",
+      "icon": "FaLinkedIn"
```
Comment (priyankarpal) [inline data/KishanGupta.json:14]: change the icon to `SiLinkedin`
```suggestion
      "icon": "SiLinkedin"
```

File: data/KishanGupta.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Kishan Gupta",
+  "type": "personal",
+  "bio": "A Full Stack Developer and Looking for contribute in Open Source Projects.",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/i_kishan_",
+      "icon": "FaTwitter"
+    }
```
Comment (priyankarpal) [inline data/KishanGupta.json:10]: Add comma 
```suggestion
    },
```

File: data/KishanGupta.json (hunk 1 above)
Comment (priyankarpal) [inline data/KishanGupta.json:15]: also here 
```suggestion
    },
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: data/KishanGupta.json [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/i-kishan",
+      "icon": "SiLinkedin"
+    },
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/kishangupta114",
+      "icon": "FaGithub"
+    },
```
Comment (SurajPratap10) [inline data/KishanGupta.json:20]:
```suggestion
    }
```

Comment (SurajPratap10) [reply data/KishanGupta.json:20]: You don't need to put  `,` here.

Comment (KishanGupta114) [reply data/KishanGupta.json:20]: Okay, Bro.. let me correct it.

Comment (KishanGupta114) [reply data/KishanGupta.json]: Bro Check it again

**Review by SurajPratap10 — APPROVED** [review]
LGTM

**Review by priyankarpal — APPROVED** [review]
Looks good to me 🎉

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
The filename must match your GitHub username.
In this case, `KishanGupta114`. Please update accordingly.

If you need any help, let us know. 🙂

**Review by md-abid-hussain — CHANGES_REQUESTED** [review]
instead of data:KishanGupta filename should be data:KishanGupta114

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**